runtime/pangolin/src/lib.rs

```diff
@@ -272,6 +272,7 @@ frame_support::construct_runtime! {
 		Assets: pallet_assets = 34,
 		Balances: pallet_balances = 5,
 		TransactionPayment: pallet_transaction_payment = 6,
+		Migrate: darwinia_migrate = 36,
```

Comment:
I think we should keep this index consistent with other runtimes, although it does not have bridge-related pallets, what do you think? @AurevoirXavier

Reply:
We can adjust this later.